_posts/2018-10-29-help-test-rust-2018.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Help test Rust 2018"
+---
+
+Back in July, we talked about ["Rust 2018"]. In short, we're adding a new
+concept to Rust, "Editions." Editions are a way of showing larger progress
+than our six-week release cycle, and will happen on a roughly three-year
+cycle. Rust 1.0 was "Rust 2015," and Rust 1.31 will be "Rust 2018."
```

Comment:
I'd love to mention not just the versions/years, but also the *themes*: stability and productivity, respectively.